scripts/sauce/sauce_connect_setup.sh

```diff
@@ -20,7 +19,15 @@ CONNECT_LOG="$LOGS_DIR/sauce-connect"
 CONNECT_STDOUT="$LOGS_DIR/sauce-connect.stdout"
 CONNECT_STDERR="$LOGS_DIR/sauce-connect.stderr"
 
-# Get Connect and start it
+# Get the appropriate URL for downloading Sauce Connect
+if [ `uname -s` = "Darwin" ]; then
```

Comment:
Wanna do Windows too for external contributors, or just ignore it until the problem comes?